### PR TITLE
Added console links for 13 resources

### PR DESCRIPTION
**Pull request by ajinkya1986**

### CODE REVIEW

File: cartography/intel/gcp/cloud_logging.py [hunk 1]
```diff
@@ -177,6 +181,7 @@ def load_logging_sinks_tx(
         sink.filter = record.filter,
         sink.destination = record.destination,
         sink.disabled = record.disabled,
+        sink.consoleLink = record.consoleLink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloud_logging.py:184]: change `sink.consoLelink` property to `sink.consolelink`

File: cartography/intel/gcp/cloudcdn.py [hunk 2]
```diff
@@ -63,6 +66,7 @@ def load_backend_buckets_tx(
         bucket.region = b.region,
         bucket.uniqueId = b.id,
         bucket.name = b.bucketName,
+        bucket.consoleLink = b.consoleLink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloudcdn.py:69]: same here change `bucket.consoLelink `property to `bucket.consolelink`

File: cartography/intel/gcp/cloudcdn.py [hunk 3]
```diff
@@ -172,6 +178,7 @@ def load_backend_services_tx(
         service.type = s.type,
         service.uniqueId = s.id,
         service.name = s.name,
+        service.consoleLink = s.consoleLink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloudcdn.py:181]: same here

File: cartography/intel/gcp/cloudmonitoring.py [hunk 4]
```diff
@@ -28,6 +29,8 @@ def get_monitoring_alertpolicies(monitoring: Resource, project_id: str) -> List[
                     policy['id'] = policy['name']
                     policy['policy_name'] = policy.get('name').split('/')[-1]
                     policy['labels'] = policy.get('userLabels', {})
+                    policy['consoleLink'] = gcp_console_link.get_console_link(project_id=project_id,\
+                        alert_policy_name=policy['name'], resource_name='cloud_monitoring_alert_policy')
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloudmonitoring.py:33]: we should pass `policy_name ` key as a `alert_policy_name `

File: cartography/intel/gcp/cloudmonitoring.py [hunk 5]
```diff
@@ -66,6 +69,7 @@ def load_monitoring_alertpolicies_tx(
         policy.region = record.region,
         policy.name = record.policy_name,
         policy.display_name = record.displayName,
+        policy.consoleLink = record.consoleLink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloudmonitoring.py:72]: Change  `policy.consoleLink` to policy.consolelink

File: cartography/intel/gcp/cloudmonitoring.py [hunk 6]
```diff
@@ -282,6 +287,7 @@ def load_monitoring_notification_channels_tx(
         channel.name = record.channel_name,
         channel.display_name = record.displayName,
         channel.enabled = record.enabled,
+        channel.consoleLink = record.consoleLink,
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloudmonitoring.py:290]: change property name format

File: cartography/intel/gcp/cloudmonitoring.py (hunk 4 above)
Comment (ajinkya1986) [reply cartography/intel/gcp/cloudmonitoring.py:33]: We are already splitting the policy name in console link project.

**Review by ganeshjagdale9545 — APPROVED** [review]
looks good

**Pull request merged**